recorded_future/plugin.spec.yaml

```
@@ -96,7 +96,7 @@ types:
       required: false
     evidenceDetails:
       title: Evidence Details
-      type: '[]evidenceDetails'
+      type: '[]evidenceDetails_obj'
```

Should revert this back
```suggestion
      type: '[]evidenceDetails'
```

---

Same as last comment